integrationTests/multiShard/block/executingMiniblocksSc_test.go

```diff
@@ -156,19 +166,23 @@ func TestProcessWithScTxsJoinAndRewardTwoNodesInShard(t *testing.T) {
 		1,
 		advertiserAddr,
 	)
+	nodeValidatorShard1.EconomicsData.SetMinGasPrice(0)
 
 	nodeProposerMeta := integrationTests.NewTestProcessorNode(
 		maxShards,
 		sharding.MetachainShardId,
 		0,
 		advertiserAddr,
 	)
+	nodeProposerMeta.EconomicsData.SetMinGasPrice(0)
```

Comment:
instead of setting the min price for all nodes, I think you can place `integrationTests.MinTxGasPrice = 0` at the top of the test and it will be set for all the instantiated nodes. This is fine because this test is not run in parallel so if you change this value it won't affect other tests.

Reply:
done.